lang/python/homeassistant/Makefile

```diff
@@ -0,0 +1,74 @@
+#
+# Copyright (C) 2019 CZ.NIC, z. s. p. o. (https://www.nic.cz/)
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=homeassistant
+PKG_VERSION:=0.93.1
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://files.pythonhosted.org/packages/source/h/homeassistant
+PKG_HASH:=ecc1e49ff2054194ce4d1b8b7f2d5052943122b8b11c46b5a833e1a66eb46079
+
+PKG_MAINTAINER:=Josef Schlehofer <pepe.schlehofer@gmail.com>
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE.md
+PKG_CPE_ID:=cpe:/a:home-assistant:home-assistant
+
+include $(INCLUDE_DIR)/package.mk
+include ../python3-package.mk
+
+define Package/$(PKG_NAME)
+  SUBMENU:=Python
+  SECTION:=lang
+  CATEGORY:=Languages
+  TITLE:=Home Assistant
+  URL:=https://www.home-assistant.io/
+  DEPENDS:= \
+    +python3-aiohttp \
+    +python3-aiohttp-cors \
+    +python3-astral \
+    +python3-async-timeout \
+    +python3-attrs \
+    +python3-bcrypt \
+    +python3-certifi \
+    +python3-cryptography \
+    +python3-ifaddr \
+    +python3-jinja2 \
+    +python3-netdisco \
+    +python3-pip \
+    +python3-pyjwt \
+    +python3-pytz \
+    +python3-pyotp \
+    +python3-slugify \
+    +python3-requests \
+    +python3-ruamel-yaml \
+    +python3-voluptuous \
+    +python3-voluptuous-serialize \
+    +python3-sqlalchemy \
+    +python3-yaml \
+    +python3-zeroconf
+  VARIANT:=python3
+endef
+
+define Package/$(PKG_NAME)/description
+  Home Assistant is a home automation platform running on Python 3.
+  It is able to track and control all devices at home and offer a platform for automating control.
+endef
+
+define Py3Package/$(PKG_NAME)/install
+	$(INSTALL_DIR) $(1)/usr/bin
+	$(CP) $(PKG_INSTALL_DIR)/usr/bin/* $(1)/usr/bin/
```

Comment:
curios: won't `/usr/bin` stuff be installed automatically?
there is some auto-magic rules that should do this; i am not recalling this 100%; but it's also fine to leave this

Reply:
They are when `install` section is not defined.

If I define `install` section like this
```
define Py3Package/$(PKG_NAME)/install
	$(INSTALL_DIR) $(1)/etc/init.d
	$(INSTALL_BIN) ./files/hass.init $(1)/etc/init.d/hass
endef
```

Then in the package are
- site-packages
- init script

Nothing more.

Reply:
oh right;
the default one does that, but you are overriding it;

thanks & sorry for the noise;
i think i did not have any coffee before i asked

Reply:
That's fine. You don't need to worry about it.